skynet/dgpu/compute.py

```diff
@@ -81,153 +69,140 @@ def prepare_params_for_diffuse(
         _params
     )
 
+_model_name: str = ''
+_model_mode: str = ''
+_model = None
```

Comment:
type again would be nice, i.e. `_model: str|None = None`

Reply:
Done in `structify` branch, created type named `Pipeline` which is `DiffusionPipeline | RealESRGANer`